push/templates/push/list.html

```diff
@@ -71,6 +71,7 @@
                             <th>{% trans "Application Name" %}</th>
                             <th>VAPID {% trans "Key" %}</th>
                             <th>VAPID {% trans "Key Status" %}</th>
+                            <th></th>
```

Comment:
Yeah, I think *Actions* is a great name. For accessibility and more, the markup should reflect the meaning of the content rather than its appearance, so I would recommend against an empty heading.